app/config/services.yml

```diff
@@ -1,83 +1,47 @@
-
-    # To inject the voter into the security layer, you must declare it as a service and tag it with security.voter.
-    # See https://symfony.com/doc/current/security/voters.html#configuring-the-voter
-    app.post_voter:
-        class: AppBundle\Security\PostVoter
+    # default configuration for services in *this* file
+    _defaults:
+        # automatically injects dependencies in your services
+        autowire: true
+        # automatically registers your services as commands, event subscribers, etc.
+        autoconfigure: true
+        # this means you cannot fetch services directly from the container via $container->get()
+        # if you need to do this, you can override this setting on individual services
         public: false
-        tags:
-            - { name: security.voter }
 
-    # Uncomment the following lines to define a service for the Post Doctrine repository.
-    # It's not mandatory to create these services, but if you use repositories a lot,
-    # these services simplify your code:
-    #
-    # app.post_repository:
-    #     class:     Doctrine\ORM\EntityRepository
-    #     factory:   ['@doctrine.orm.entity_manager', getRepository]
-    #     arguments: [AppBundle\Entity\Post]
-    #
-    # // traditional code inside a controller
-    # $entityManager = $this->getDoctrine()->getManager();
-    # $posts = $entityManager->getRepository('AppBundle:Post')->findAll();
-    #
-    # // same code using repository services
-    # $posts = $this->get('app.post_repository')->findAll();
+    # makes classes in src/AppBundle available to be used as services
+    # this creates a service per class whose id is the fully-qualified class name
+    AppBundle\:
+        resource: '../../src/AppBundle/*'
+        # you can exclude directories or files
+        # but if a service is unused, it's removed anyway
+        exclude: '../../src/AppBundle/{Controller,Entity,Repository}'
+
+    # controllers are imported separately to make sure they're public
+    # and have a tag that allows actions to type-hint services
+    AppBundle\Controller\:
+        resource: '../../src/AppBundle/Controller'
+        public: true
+        tags: ['controller.service_arguments']
+
+    # Autowiring can't guess the constructor arguments that are not type-hinted with
+    # classes (e.g. container parameters) so you must define those arguments explicitly
+    AppBundle\Command\ListUsersCommand:
+        arguments:
+            $emailSender: '%app.notifications.email_sender%'
+
+    AppBundle\Twig\AppExtension:
+        arguments:
+            $locales: '%app_locales%'
+
+    AppBundle\EventListener\CommentNotificationSubscriber:
+        arguments:
+            $sender: '%app.notifications.email_sender%'
+
+    AppBundle\EventListener\RedirectToPreferredLocaleSubscriber:
+        arguments:
+            $locales: '%app_locales%'
+            $defaultLocale: '%locale%'
+
+
+    Twig\Extensions\IntlExtension: ~
```

Comment:
Ah, what about adding a small note to this service definition where say it's used by `localizeddate` filter in Twig templates?